taimen.json

```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "Sudhakar Jha (HtcDesire2)",
+        "oem": "Google",
+        "device": "Pixel 2 XL",
+        "filename": "crDroidAndroid-12.1-20220329-taimen.v8.3.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/taimen/8.x/crDroidAndroid-12.1-20220329-taimen.v8.3.zip/download",
+        "timestamp": 1648543159,
+        "md5": "e23af944bf85c0f019002fb3c29bdfc0",
+        "sha256": "952d705f37230c3836b1282c123876a1e6f0e7d66e7c27c178b9ed50ab0ae28c",
+        "size": 1136423444,
+        "version": "8.3",
+        "buildtype": "Monthly",
+        "forum": "https://forum.xda-developers.com/t/rom-12-1-taimen-crdroid-8-3-with-gapps-unofficial-30-03-2022.4400165/",
+        "gapps": "",
+        "firmware": "https://dl.google.com/dl/android/aosp/taimen-rp1a.201005.004.a1-factory-2f5c4987.zip",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "https://dl.twrp.me/taimen/",
+        "paypal": "",
+        "telegram": "https://t.me/crDroidPixel",
+        "dt": "https://github.com/crdroidandroid/device_google_taimen",
```

Comment:
push dt

Reply:
Done. Forked to crdroidandroid/device_google_taimen